bundles/org.openhab.binding.renault/src/main/java/org/openhab/binding/renault/internal/api/Car.java

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.renault.internal.api;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+
+/**
+ * MyRenault registered car for parsing HTTP responses and collecting data and
+ * information.
+ *
+ * @author Doug Culnane - Initial contribution
+ */
+@NonNullByDefault
+public class Car {
+
+    private final Logger logger = LoggerFactory.getLogger(Car.class);
+
+    public boolean disableLocation = false;
+    public boolean disableBattery = false;
+    public boolean disableCockpit = false;
+    public boolean disableHvac = false;
```

Comment:
Please use private members and add getters/setters
No need to set the member to false as it is the implicit default.

Reply:
Done.